src/request/request-tab.js

```diff
@@ -34,7 +34,8 @@ module.exports = {
 // TODO: Need to come up with a better self registration process
 require('./components/request-detail-panel-execution');
 var request = require('./components/request-detail-panel-request');
-require('./components/request-detail-panel-logging');
+var LoggingComponent = require('./components/request-detail-panel-logging').Logging;
+var LoggingViewModel = require('./view-models/LoggingViewModel').LoggingViewModel;
```

Comment:
I'll concede the point about naming--it's unfortunate that React uses the terribly-generic term "component" for what everyone else would call a "view".  I'd be fine renaming the folder to `component-model` and types `IComponentModel`-based.

I disagree that the view models are *for* the views, as opposed to a component in its own right that exists to encapsulate and coordinate the interaction between the user, the application, and its data.    It's in that sense that I prefer to keep view models as peers of the views.  (Whether both should be at a peer level with the rest of the application components is another question.)